test/isNotFloat.js

```diff
@@ -3,19 +3,18 @@ import MAX_SAFE_INTEGER from '../src/internal/polyfills/Number.MAX_SAFE_INTEGER'
 import MIN_SAFE_INTEGER from '../src/internal/polyfills/Number.MIN_SAFE_INTEGER';
 import eq from './shared/eq';
 
-
-describe('isNotFloat', function () {
-  it('tests a value for complement of a `float`', function () {
+describe('isNotFloat', function() {
+  it('tests a value for complement of a `float`', function() {
     eq(RA.isNotFloat(0), true);
     eq(RA.isNotFloat(1), true);
     eq(RA.isNotFloat(-100000), true);
     eq(RA.isNotFloat(MAX_SAFE_INTEGER), true);
     eq(RA.isNotFloat(MIN_SAFE_INTEGER), true);
-    eq(RA.isNotFloat(5e+0), true);
+    eq(RA.isNotFloat(5), true);
 
     eq(RA.isNotFloat(0.1), false);
     eq(RA.isNotFloat(Math.PI), false);
-    eq(RA.isNotFloat(5.56789e+0), false);
+    eq(RA.isNotFloat(5.56789), false);
```

Comment:
the scientific notation has been dropped here, but was here by design, can we ignore here ?